app/assets/stylesheets/common/base/activation.scss

```diff
@@ -1,73 +1,70 @@
 // Styles used before the user is logged into discourse. For example, activating their
 // account or changing their email.
 
-.account-activation-page {
-  .desktop-view & {
-    background: var(--primary-very-low);
-  }
-  #main-outlet-wrapper {
-    display: block;
-    .sidebar-wrapper {
-      display: none;
-    }
-  }
-  .header-sidebar-toggle {
-    display: none;
-  }
+.account-created-page,
+.activate-account-page {
+  background: var(--secondary);
+
   #main-outlet {
     padding: 0;
     height: 100%;
   }
-  #simple-container {
+}
+
+.activate-account-page .alert-error {
+  margin: 1em;
+}
+
+.account-created,
+.activate-account {
+  max-width: 500px;
+  padding: 2rem 3rem;
+  background: var(--secondary);
+  box-shadow: var(--shadow-menu-panel);
+  margin: 10vh auto 1em auto;
```

Comment:
A neat trick to position something centrally is this:
```
position: absolute;
top: 50%;
left: 50%;
transform: translate(-50%, -50%);
```

Reply:
Those styles are meant to match the format in https://github.com/discourse/discourse/blob/92f03a1108e4a90a53394323f439ec36fc36aa54/app/assets/stylesheets/desktop/invite-signup.scss#L5-L14

We were using `transform` but we changed it in
https://github.com/discourse/discourse/commit/7639591552d17f6b42884f2bf4a32db2b9875c11, not sure why tho

Reply:
Hm odd, those margins seems so arbitrarily chosen to me.